core/actionProxy/Dockerfile

```diff
@@ -1,13 +1,14 @@
-# Dockerfile for docker skeleton (useful for running blackbox binaries, scripts, or python actions).
-FROM python:2.7.12-alpine
+# Dockerfile for docker skeleton (useful for running blackbox binaries, scripts, or Python 3 actions).
+FROM python:3.6.0-alpine
 
 # Upgrade and install basic Python dependencies
 RUN apk add --no-cache bash \
  && apk add --no-cache --virtual .build-deps \
         bzip2-dev \
         gcc \
         libc-dev \
-  && pip install --no-cache-dir gevent==1.1.2 flask==0.11.1 \
+  && pip install --upgrade pip setuptools \
```

Comment:
There are [frequent ongoing changes](https://github.com/pypa/setuptools/releases) in both pip and in setuptools to make sure that Python packaging is ready for the shift for [PyPI](https://pypi.python.org/pypi) to its more modern replacement, [warehouse](https://pypi.org/).  We want to ensure that dockerskeleton can properly install both legacy packages like twisted and more modern packages that are built for warehouse.  This will become even more important as features like PR 1940 will allow action developers to layer their own packages (or package versions) on top of the runtime.